src/backend/sev/snp/launch/mod.rs

```diff
@@ -18,6 +18,11 @@ use std::os::unix::io::AsRawFd;
 
 use bitflags::bitflags;
 
+use kvm_bindings::bindings::kvm_enc_region;
+use kvm_ioctls::Kvm;
+
+use libc;
```

Comment:
https://github.com/enarx/enarx/pull/1247#discussion_r788699935

Reply:
I've now fixed this!